gems/aws-sigv4/sig/error.rbs

```diff
@@ -0,0 +1,13 @@
+module Aws
+  module Sigv4
+    module Errors
+      class MissingCredentialsError < ArgumentError
+        def initialize: (?String? msg) -> void
```

Comment:
I think this is technically correct (since you could explicitly pass nil here), but I think we may want to just type it as `?String`

Reply:
Fixed!
5c6600f4cf845af9eedd8ebee14c1c8175dcd544